cmd/minishift/cmd/start_preflight.go

```diff
@@ -49,6 +49,7 @@ const (
 // preflightChecksAfterStartingHost is executed before the startHost function.
 func preflightChecksBeforeStartingHost() {
 	driverErrorMessage := "See the 'Setting Up the Driver Plug-in' topic (https://docs.openshift.org/latest/minishift/getting-started/setting-up-driver-plugin.html) for more information"
+	vBoxErrorMessage := "See the 'Installing Minishift' topic (https://docs.openshift.org/latest/minishift/getting-started/installing.html#install-prerequisites) for more information"
```

Comment:
s/Installing Minishift/Installing prerequisites for Minishift/

Reply:
Making the changes.